gnome-initial-setup/pages/account/gis-account-page-local.c

```diff
@@ -417,6 +420,21 @@ get_default_avatar_image (void)
   return default_avatar;
 }
 
+static gboolean
+show_eula_dialog (GtkLabel    *label,
+                  const gchar *uri,
+                  gpointer     user_data)
+{
+  GtkWindow *toplevel = GTK_WINDOW (gtk_widget_get_toplevel (GTK_WIDGET (label)));
+  GisAccountPageEulaDialog *dialog = g_object_new (GIS_TYPE_ACCOUNT_PAGE_EULA_DIALOG,
```

Comment:
Personally I'd prefer if this `g_object_new` lived in a `gis_account_page_eula_dialog_new ()` function defined in that .c/.h but hey ho.